### ISSUE TITLE
Question: Get parsed member list from a template?

### ISSUE THREAD

**Issue by marlon-tucker** [sebastienros/fluid#492]
Hello,

Is there any way to get a list of members contained in a template.

For example, the template `Hello {{ Firstname }} {{ Lastname }}`, I need a way to get the strings `"Firstname"` and `"Lastname"`.

If there's no built in method, would you consider having a AST visitor pattern in the project? A visitor is heavily tied to the AST so it makes sense for it to be in the same project.

Thanks

Marlon

**Comment by hishamco** [comment]
Did you dig into a `MemberAccessor`

**Comment by hishamco** [comment]
There's few typed for access members, please check https://github.com/sebastienros/fluid/tree/e6156835028b91e71634311076060ee75bfe5f6b/Fluid probably one of them achieve what you want

I'm busy with another PR, if I got a time I may look too

**Comment by sebastienros** [comment]
I agree there is a need for something like a visitor. I have had the need recently to be able to find in a node has subnodes of a specific type (example: in a for loop, is there any tag that uses the `forloop` object).

**Comment by caspertl** [comment]
I did this with custom code. But this wasn't easy :D look at 

               var template = FluidParser.Parse(content);
               **var statements = ((Fluid.Parser.FluidTemplate)template).Statements;**

**Comment by sebastienros** [comment]
There has been full visitor support for some time now, closing
